ansible/roles/host-ocp4-provisioner/tasks/azure_prereqs.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Install required packages for Azure CLI
+  package:
+    name:
+    - libffi
+    - openssl
+    - openssl-devel
+    - python3
```

Comment:
We'll see if this works. In general we try to do all Python stuff these days in virtualenvs.

Reply:
its a dedicated bastion host for just installing the OCP4 cluster, so I don't care about messing with other workloads so I just copied the logic from the osp_prereqs.yml